src/components/KTabs/KTabs.vue

```diff
@@ -8,18 +8,18 @@
         v-for="(tab, i) in tabs"
         :id="`${tab.hash.replace('#','')}-tab`"
         :key="tab.hash"
-        :aria-controls="hasPanels ? `panel-${i}` : undefined"
-        :aria-selected="hasPanels ? (activeTab === tab.hash ? 'true' : 'false') : undefined"
         class="tab-item"
-        :class="{ active: activeTab === tab.hash }"
-        :role="hasPanels ? 'tab' : undefined"
-        tabindex="0"
-        @click="handleTabChange(tab.hash)"
-        @keydown.enter.prevent="handleTabChange(tab.hash)"
-        @keydown.space.prevent="handleTabChange(tab.hash)"
+        :class="{ active: activeTab === tab.hash, disabled: tab.disabled }"
       >
         <div
-          :class="['tab-link', { 'has-panels': hasPanels }]"
+          :aria-controls="hasPanels ? `panel-${i}` : undefined"
+          :aria-selected="hasPanels ? (activeTab === tab.hash ? 'true' : 'false') : undefined"
+          class="tab-link"
+          :role="hasPanels ? 'tab' : undefined"
+          :tabindex="tab.disabled ? '-1' : '0'"
+          @click="handleTabChange(tab.hash)"
```

Comment:
suggestion: is there any reason not to use an anchor tag for the clickable element instead of a `div`?

Reply:
Anchor tag requires `href` attribute to be provided at all times. Considering that usage of this component doesn't always imply using hash in the route, I think it's not ideal. Also makes things trickier with disabled item.

Reply:
I'm not sure I understand -- a clickable div isn't semantically correct.

> Anchor tag requires href attribute to be provided at all times.

`<a href="#" @click.prevent="doSomething" />`

This has nothing to do with a hash in the route?

> Also makes things trickier with disabled item

```scss
a.disabled {
  cursor: not-allowed;
}
```

In the `doSomething` method, check if disabled. If true, exit.

Reply:
Ah you're right, `.prevent` will prevent it from adding the `#` in the route. Changed it to anchor tag.